common/changes/office-ui-fabric-react/textfield-icon-required_2017-10-24-16-24.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "New styling for required textfield with an icon but no label.",
```

Comment:
TextField/DatePicker: The required astrisk is now more correctly positioned.

Reply:
Also make sure to check variants. Are we ok with multiline?

Reply:
It actually looks like the icon positioning was / is still off. I'll check in with Ben and add it to my backlog is it looks like more than a super simple fix.